src/openrct2-ui/windows/GuestList.cpp

```diff
@@ -948,18 +944,21 @@ static void window_guest_list_find_groups()
     _window_guest_list_num_groups = 0;
 
     // Set all guests to unassigned
-    FOR_ALL_GUESTS (spriteIndex, peep)
-        if (peep->outside_of_park == 0)
-            peep->flags |= SPRITE_FLAGS_PEEP_VISIBLE;
-
+    {
+        auto list = EntityList<Guest>(SPRITE_LIST_PEEP);
+        for (auto peep : list)
+            if (peep->outside_of_park == 0)
```

Comment:
```suggestion
            if (!peep->outside_of_park)
```

Reply:
Outside of park is not a bool (yet).

Reply:
It is still used as such, though.

Reply:
I'd prefer to address this all as one in a later PR.

Reply:
OK, but could you create an issue then?